### PR TITLE
Add ability to process multiple demo files at once

### PR DESCRIPTION
**Pull request by billfreeman44**

### CODE REVIEW

**Comment by billfreeman44** [comment]
I also removed that keyboard wait at the end. I can add that back in if you want :)

File: main.go [hunk 1]
```diff
@@ -264,16 +266,16 @@ type playerStats struct {
...
-
-	//for concurrency
-	//for i := 0; i < 10; i++ {
-	//	go processDemo("faceit1.dem")
-	//}
-
-	processDemo("oc3.dem")
-
-	var input string
-	fmt.Scanln(&input)
```
Comment (cokenyo) [inline main.go]: Leave this in please.

**Review by cokenyo — CHANGES_REQUESTED** [review]
See comments.

File: main.go (hunk 1 above)
Comment (billfreeman44) [reply main.go]: NO U

**Review by cokenyo — APPROVED** [review]
Thanks!

**Pull request merged**